tests/unit_tests/kd_ema_test.py

```diff
@@ -0,0 +1,140 @@
+import unittest
+from super_gradients.training.sg_model import SgModel
+from super_gradients.training.kd_model.kd_model import KDModel
+import torch
+from super_gradients.training.utils.utils import check_models_have_same_weights
+from super_gradients.training.datasets.dataset_interfaces.dataset_interface import ClassificationTestDatasetInterface
+from super_gradients.training.metrics import Accuracy
+from super_gradients.training.losses.kd_losses import KDLogitsLoss
+
+
+class KDModelTest(unittest.TestCase):
```

Comment:
The PR looks good!

Lets just add the unit test to the test_suite_tunner so it would be triggered in the CI and were good to go.